configmap/init/sql/repl/privileges.tpl.sql

```diff
@@ -0,0 +1,6 @@
+CREATE USER 'qsreplica'@'%' IDENTIFIED BY '<MYSQL_REPLICA_PASSWORD>';
+GRANT ALL ON qservReplica.* TO 'qsreplica'@'%';
```

Comment:
All privileges are only required for the host(s) where the Replication Controllers run. Tis is probably fine to allow any host to assume all privileges for as long as the database service is not visible from a specific K8 deployment (application).